e2e/libs/backupstore.py

```diff
@@ -0,0 +1,736 @@
+import os
+import time
+import base64
+import hashlib
+import json
+import subprocess
+from minio import Minio
+from minio.error import ResponseError
+from urllib.parse import urlparse
+
+LONGHORN_NAMESPACE = "longhorn-system"
+
+SETTING_BACKUP_TARGET = "backup-target"
+SETTING_BACKUP_TARGET_CREDENTIAL_SECRET = "backup-target-credential-secret"
+SETTING_BACKUPSTORE_POLL_INTERVAL = "backupstore-poll-interval"
+
+BACKUPSTORE_BV_PREFIX = "/backupstore/volumes/"
+BACKUPSTORE_LOCK_DURATION = 150
+
+TEMP_FILE_PATH = "/tmp/temp_file"
+
+RETRY_COUNT = 300
+RETRY_INTERVAL = 1
```

Comment:
This can be refined later if it actually has the situation that deleting backup volumes is unexpected time-consuming. If it's observed, it can be refined.